com.unity.netcode.gameobjects/Tests/Runtime/com.unity.netcode.runtimetests.asmdef

```diff
@@ -11,11 +11,12 @@
         "Unity.Multiplayer.Tools.MetricTypes",
         "Unity.Multiplayer.Tools.NetStats",
         "Unity.Netcode.Adapter.UTP",
-        "ClientNetworkTransform"
+        "ClientNetworkTransform",
+        "Unity.Netcode.TestHelpers.Runtime"
     ],
     "includePlatforms": [],
     "excludePlatforms": [],
-    "allowUnsafeCode": false,
+    "allowUnsafeCode": true,
```

Comment:
what's the unsafe code getting into this asmdef now?

Reply:
Because of the serialization and fastbuffer related tests.